src/mgr/DaemonServer.cc

```diff
@@ -2331,6 +2334,7 @@ void DaemonServer::adjust_pgs()
 			       << " pgs in " << pg_state_string(j.first)
 			       << dendl;
 		      active = false;
+                      break;
```

Comment:
i think leaving the break out is better; the log then shows all pg states that aren't active

Reply:
eh, this is good too.